src/runtime-rs/crates/hypervisor/src/qemu/inner.rs

```diff
@@ -65,15 +64,11 @@ impl QemuInner {
         info!(sl!(), "Starting QEMU VM");
         let netns = self.netns.clone().unwrap_or_default();
 
+        // CAUTION: since 'cmdline' contains file descriptors that have to stay
+        // open until spawn() is called to launch qemu later in this function,
+        // 'cmdline' has to live at least until spawn() is called
```

Comment:
This would obviously be the case if ownership goes to the  `command` variable using https://crates.io/crates/command-fds and this comment wouldn't be needed.

Possible improvement in a follow-up PR. In anycase, the change does the job.

Reply:
Frankly, I only added the comment because I thought you'd appreciate it. ;-)

This was discussed off-line so just to summarise my stance for the public record: for this to become a problem, someone would need to drop `cmdline` and before doing so, store the `cmdline`'s output somewhere.  I don't quite see how this could happen inadvertently, seeing as someone would need to work to make the problem happen.

Also, if that did actually happen somehow, the problem would be very conspicuous since it would break everything (among other problems, it would prevent runtime-rs from talking with kata-agent so launch would fail immediately and consistently).

All in all, if I'm wrong and this problem actually materialises I'll be very curious to know how it happened.